dpe/src/commands/certify_key.rs

```diff
@@ -109,10 +109,12 @@ impl CommandExecution for CertifyKeyCmd {
         let mut subj_serial = [0u8; DPE_PROFILE.get_hash_size() * 2];
         env.crypto
             .get_pubkey_serial(DPE_PROFILE.alg_len(), &pub_key, &mut subj_serial)?;
+        // The serial number of the subject can be at most 64 bytes
+        let truncated_subj_serial = &subj_serial[..64];
```

Comment:
When we go to enable these tests in caliptra-sw, I think all the other certs will fail this check: https://github.com/chipsalliance/caliptra-sw/blob/b305b01e4cfd843e7f48151726f813b8b5d3e91e/x509/build/csr.rs#L105-L107

I wonder if we should just explicitly disable this check in the linter since the ROM cert chain won't comply.

We could open an issue in caliptra-sw to potentially fix for a 1.x release

Reply:
Added this check to the exclusion list.

Reply:
Opened https://github.com/chipsalliance/caliptra-sw/issues/1372 to track